install_config/install/advanced_install.adoc

```diff
@@ -390,8 +390,23 @@ details.
 |`openshift_clusterid`
 |This variable is a cluster identifier unique to the AWS Availability Zone. Using this avoids potential issues in Amazon Web Service
 (AWS) with multiple zones or multiple clusters. See xref:../../install_config/configuring_aws.adoc#aws-cluster-labeling[Labeling Clusters for AWS] for details.
+
+|`openshift_image_tag`
+|Use this variable to specify a container image tag to install or configure.
+
+|`openshift_pkg_version`
+|Use this variable to specify a rpm version to install or configure.
+
 |===
 
+[WARNING]
+====
+Be careful with modifying the `openshift_image_tag` or the `openshift_pkg_version` variables after the cluster is set up, as it could trigger an upgrade and result in downtime.
```

Comment:
Maybe an if, then statement here?
If you modify the `openshift_image_tag` or the `openshift_pkg_version` variables after the cluster is set up, then an upgrade can be triggered, resulting in downtime.